<?xml version="1.0"?>
<sdf version="1.9">
    <model name="mbzirc_planar_lidar">
        <link name="sensor_link">
            <inertial>
                <mass>0.005</mass>
                <inertia>
                    <ixx>8.33e-06</ixx>
                    <ixy>0</ixy>
                    <ixz>0</ixz>
                    <iyy>8.33e-03</iyy>
                    <iyz>0</iyz>
                    <izz>8.33e-03</izz>
                </inertia>
            </inertial>
            <sensor name="lidar" type="gpu_ray">
                <update_rate>30</update_rate>
                <lidar>
                    <scan>
                        <horizontal>
                            <samples>720</samples>
                            <resolution>1</resolution>
                            <min_angle>-2.3562</min_angle>
                            <max_angle>2.3562</max_angle>
                        </horizontal>
                    </scan>
                    <range>
                        <min>0.04</min>
                        <max>30</max>
                        <resolution>0.01</resolution>
                    </range>
                    <noise>
                        <type>gaussian</type>
                        <mean>0</mean>
                        <stddev>0.01</stddev>
                    </noise>
                    <visibility_mask>55</visibility_mask>
                </lidar>
            </sensor>
        </link>
        <frame name="mount_point"/>
    </model>
</sdf>
